.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,239 @@
+# - PACT_BROKER_PASSWORD - the Pact Broker password
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+  push:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+  workflow_dispatch:
+    inputs:
+      pb-event-type:
+        description: 'the Pact Broker event type that triggers this workflow'
+        required: true
+        type: string
+      consumer-name:
+        description: 'the consumer name'
+        required: true
+        type: string
+      consumer-version-number:
+        description: 'the version number of the most recent consumer version associated with the pact content'
+        required: true
+        type: string
+      provider-version-number:
+        description: 'the provider version number for the verification result'
+        required: false
+        type: string
+      consumer-version-tags:
+        description: 'the list of tag names for the most recent consumer version associated with the pact content, separated by ", "'
+        required: true
+        type: string
+      consumer-version-branch:
+        description: 'the name of the branch for most recent consumer version associated with the pact content'
+        required: true
+        type: string
+      provider-version-branch:
+        description: 'the name of the branch for the provider version associated with the verification result'
+        required: false
+        type: string
+      consumer-labels:
+        description: 'the list of labels for the consumer associated with the pact content, separated by ", "'
+        required: false
+        type: string
+      provider-labels:
+        description: 'the list of labels for the provider associated with the pact content, separated by ", "'
+        required: false
+        type: string
+      pact-url:
+        description: 'the "permalink" URL to the newly published pact (the URL specifying the consumer version URL, rather than the "/latest" format'
+        required: true
+        type: string
+
+env:
+  CAN_I_DEPLOY_RUN_NAME: 'can-i-deploy-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
+  PACT_BROKER_URL: pact-broker.dsp-eng-tools.broadinstitute.org
+
+jobs:
+
+  bump-check:
+    runs-on: ubuntu-latest
+    outputs:
+      is-bump: ${{ steps.skiptest.outputs.is-bump }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Skip version bump merges
+        id: skiptest
+        uses: ./.github/actions/bump-skip
+        with:
+          event-name: ${{ github.event_name }}
+
+  # We only need a new version tag when this workflow is triggered by opening, updating a PR or PR merge.
+  # When triggered by a Pact Broker webhook, the provider version (GIT hash or release tag)
+  # is already included in the payload, then a new version tag wouldn't be needed.
+  regulated-tag-job:
+    needs: [ bump-check ]
+    if: ${{ needs.bump-check.outputs.is-bump == 'no' }}
+    uses: ./.github/workflows/tag.yml
```

Comment:
Let's update `tag.yml` to support running this job synchronously (workflow_call).

```
name: Tag
on: 
  workflow_dispatch:
  workflow_call:
    inputs:
      ref:
        description: "The branch, tag or SHA to checkout"
        default: ''
        required: false
        type: string
      dry-run:
        description: "Determine the next version without tagging the branch. The workflow can use the outputs new_tag and tag in subsequent steps. Possible values are true and false (default)"
        default: false
        required: false
        type: string
    secrets:
      BROADBOT_TOKEN:
        required: true

jobs:
  tag-job:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout current code
        uses: actions/checkout@v3
        with:
          ref: ${{ inputs.ref || '' }}
          token: ${{ secrets.BROADBOT_TOKEN }} # this allows the push to succeed later
      - name: Bump the tag to a new version
        # https://github.com/DataBiosphere/github-actions/tree/master/actions/bumper
        uses: databiosphere/github-actions/actions/bumper@bumper-0.3.0
        id: tag
        env:
          GITHUB_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
          DEFAULT_BUMP: minor
          RELEASE_BRANCHES: dev
          VERSION_FILE_PATH: build.gradle
          VERSION_LINE_MATCH: "^\\s*version\\s*=\\s*'.*'"
          VERSION_SUFFIX: SNAPSHOT
          DRY_RUN: ${{ inputs.dry-run || false }}
```